src/patch_parser.py

```diff
@@ -11,7 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
+import dataclasses
```

Comment:
very minor nit: can we add a \n before this so there's a separation between license comment and the code?